packages/react/src/components/SuiteHeader/SuiteHeader.test.jsx

```diff
@@ -235,6 +244,122 @@ describe('SuiteHeader', () => {
     userEvent.click(screen.getByRole('button', { name: 'closes notification' }));
     expect(screen.queryByRole('alert')).toBeNull();
   });
+  it('active user does not see idle logout confirmation dialog', () => {
+    render(<SuiteHeader {...commonProps} idleTimeoutData={idleTimeoutDataProp} />);
+    // Simulate a timestamp cookie that is in the future
+    Object.defineProperty(window.document, 'cookie', {
+      writable: true,
+      value: `${idleTimeoutDataProp.cookieName}=${Date.now() + 1000}`,
+    });
```

Comment:
Should add a reset in afterEach, so that these cookie values are cleared out after each test just to prevent any unexpected side effects.

Reply:
Thanks. I've just added a change that saves `window.document.cookie` before each test and restores it after each test.